pubspec.yaml

```diff
@@ -10,9 +10,9 @@ dependencies:
   args: ^0.13.0
   cli_util: ^0.0.1
   html: ^0.12.1
-  http: ^0.11.2
-  intl: ^0.12.0
-  logging: '>=0.11.1 <0.12.0'
+  http: '>=0.11.0'
```

Comment:
Doesn't this mean that we're saying we'll work with any future version? That seems risky.

I thought setting an upper bound was a best practice?